docs/sources/panels-visualizations/visualizations/candlestick/index.md

```diff
@@ -21,12 +21,53 @@ weight: 100
 
 # Candlestick
 
-The candlestick visualization allows you to visualize data that includes a number of consistent dimensions focused on price movement. The candlestick visualization includes an Open-High-Low-Close (OHLC) mode, as well as support for additional dimensions based on time series data.
-
-{{< figure src="/static/img/docs/candlestick-panel/candlestick-panel-8-3.png" max-width="1200px" caption="Candlestick visualization" >}}
+The candlestick visualization allows you to visualize data that includes a number of consistent dimensions focused on price movements, such as stock prices. The candlestick visualization includes an [Open-High-Low-Close (OHLC) mode](#open-high-low-close), as well as support for additional dimensions based on time series data.
 
 Candlestick visualizations build upon the foundation of the [time series visualization][] and include many common configuration settings.
 
+For example, you can use a candlestick if you want to visualize at a glance how the price moved over time, whether it went up, down, or stayed the same, and how much it fluctuated:
+
+{{< figure src="/static/img/docs/candlestick-panel/candlestick-panel-8-3.png" max-width="1065px" alt="A candlestick visualization" >}}
+
+Each candlestick is represented as a rectangle, referred to as the "candlestick body". The candlestick body displays the opening and closing prices during a time period. The green candlesticks represent when the price appreciated while the red candlesticks represent when the price depreciated. The lines sticking out the candlestick body are referred to as "wicks" or "shadows", which represent the highest and lowest prices during the time period.
+
+Use a candlestick when you need to:
+
+- Monitor and identify trends in price movements of specific assets such as stocks, currencies, or commodities.
+- Analyze any volatility in the stock market.
+- Provide data analysis to help with trading decisions.
+
+## Configure a candlestick
+
+<!-- video TBA here -->
+
+{{< docs/play title="Candlestick Visualizations in Grafana" url="https://play.grafana.org/d/candlestick/candlestick" >}}
+
+## Supported data formats
+
+The candlestick visualization works best if you have the price movements of the asset that you are interested in tracking. The data must include:
+
+- **Timestamps** - The time period for when each price movement occurred.
```

Comment:
This is a bit unclear. My understanding is each individual price movement has a specific timestamp.
```suggestion
- **Timestamps** - The time at which each price movement occurred.
```

Reply:
Yes, your understanding is correct! Thank you for correcting it :)